apps/src/music/views/MusicLabView.tsx

```diff
@@ -119,6 +121,25 @@ const MusicLabView: React.FunctionComponent<MusicLabViewProps> = ({
     }
   }, [blocklyWorkspace, isStartMode, levelData]);
 
+  // Use the Lab2 generic prompt for Blockly prompt dialogs.
+  const genericPrompt = useCallback(
```

Comment:
super nit: maybe `showGenericPrompt` or `showPrompt` since this is a function and not the prompt component itself?

Reply:
Updated!